src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/interior_data_cache_test.cc

```diff
@@ -34,67 +34,66 @@
 
 namespace rc_rpc_plugin_test {
 
-class InteriorDataCacheTest : public ::testing::Test  {
-};
+class InteriorDataCacheTest : public ::testing::Test {};
 
-TEST_F(InteriorDataCacheTest, InteriorDataCacheDoesNotContainRandomDataInitialy) {
-    const rc_rpc_plugin::InteriorDataCacheImpl cache;
-    std::string module_type_key = "random_module_type";
-    EXPECT_FALSE(cache.Check(module_type_key));
-    auto retrived_data = cache.Retrive(module_type_key);
-    EXPECT_EQ(retrived_data.getType(), smart_objects::SmartType_Null);
+TEST_F(InteriorDataCacheTest,
+       InteriorDataCacheDoesNotContainRandomDataInitialy) {
+  const rc_rpc_plugin::InteriorDataCacheImpl cache;
+  std::string module_type_key = "random_module_type";
+  EXPECT_FALSE(cache.Check(module_type_key));
+  auto retrived_data = cache.Retrive(module_type_key);
```

Comment:
@LuxoftAKutsan error in variable name

Reply:
@AByzhynar fixed 3258c5b2c9f70cc3c7c4997882b94a8f6ab9f600